feat(containerSample): add swap and storage metrics (#1936)
newrelic/docs-website: src/data-dictionary/events/ContainerSample/memoryUsageBytes.md

  - ContainerSample
---

The amount of memory the container is using. This memory doesn't include swap.

> **Breaking change**: This metric includes memory + swap on versions v1.5.0 or higher of `nri-docker`.

Reviewer: ```suggestion
**Breaking change**: This metric used to include swap memory `nri-docker` v1.5.0 and earlier (Agent `1.16.x` and earlier).
```

⬆️ Users probably do not know which version of `nri-docker` they have since it is bundled with the agent.

> The amount of memory the container is using. This memory doesn't include swap.
> 
> **Breaking change**: This metric includes memory (including swap) on versions v1.5.0 or higher of `nri-docker`.

Reviewer: Users probably do not know which version of `nri-docker` they have since it is bundled with the agent.

```suggestion
The amount of memory the container is using. This memory doesn't include swap.

**Breaking change**: This metric includes memory (including swap) on versions v1.5.0 or higher of `nri-docker`.
```
```suggestion
The amount of memory the container is using. This memory does not include include swap usage.

**Breaking change**: This metric used to include swap memory in `nri-docker` v1.5.0 and earlier (Infrastructure Agent `1.16.x` and earlier). The amount of used memory including swap is now reported as `memorySwapUsageBytes`.
```
